Common/Data/Auxiliary/MapFile.cs

```diff
@@ -131,6 +143,16 @@ public static MapFile Read(string permtick, string market)
             return new MapFile(permtick, MapFileRow.Read(permtick, market));
         }
 
+        /// <summary>
+        /// Writes the map file to a CSV file
+        /// </summary>
+        /// <param name="market">The market to save the MapFile to</param>
+        public void WriteToCsv(string market) 
+        {
+            var filePath = GetMapFilePath(Permtick, market);
```

Comment:
Maybe we can heck if the folder exist first.

Reply:
What kind of behavior should we be aiming for here? I'm thinking about writing an error message to the log and returning with no write attempted if the directory can't be found

Reply:
I think that with a simple `Directory.Create(path)` is enough. If there is no folder it'll be created, if it exists, nothing happens